### PR TITLE
Nerfs Lilly Trilby with a new jumpsuit that kills her in one bullet shot of almost any kind

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
[Changelogs]
Poorly adds in a new suit and hat for Lilly Trilby to be one shot in
inb4 I take time to make it cursed onto That hecking Masked Cat 
[why]
Poorly Nerfs Nerd Lilly Trilby cat - The Catto

### CODE REVIEW

File: modular_citadel/code/modules/client/loadout/__donator.dm [hunk 1]
```diff
@@ -367,4 +367,18 @@ datum/gear/darksabresheath
...
-	ckeywhitelist = list("twilightic")
\ No newline at end of file
+	ckeywhitelist = list("twilightic")
+
+/datum/gear/telegramsuit
+	name = "Singing Telegram Suit"
+	category = SLOT_IN_BACKPACK
+	cost = 0
+	path = /obj/item/clothing/head/telegramhat
+	ckeywhitelist = list("TrilbySpaseClone")
```
Comment (Ghommie) [inline modular_citadel/code/modules/client/loadout/__donator.dm]: wasn't it `TrilbySpaceClone` ? Make sure you wrote down your own ckey correctly.

File: modular_citadel/code/modules/client/loadout/__donator.dm [hunk 2]
```diff
@@ -367,4 +367,18 @@ datum/gear/darksabresheath
...
+
+/datum/gear/telegramhat
+	name = "Singing Telegram Hat"
+	category = SLOT_IN_BACKPACK
+	cost = 0
+	path = /obj/item/clothing/under/telegramhat
+	ckeywhitelist = list("TrilbySpaseClone")
```
Comment (Ghommie) [inline modular_citadel/code/modules/client/loadout/__donator.dm]: idem

File: modular_citadel/code/modules/custom_loadout/custom_items.dm [hunk 3]
```diff
@@ -439,3 +436,24 @@
...
+	alternate_worn_icon = 'icons/mob/custom_w.dmi'
+	desc = "A red fez like hat, has a yellow lining on the side."
+	item_state = "telegramhat"
+	icon_state = "telegramhat"
+	armor = list("melee" = 0, "bullet" = -500, "laser" = 0, "energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 0, "acid" = 0) //Dont shoot the messager!
+	mutantrace_variation = NO_MUTANTRACE_VARIATION
+
+/obj/item/clothing/under/telegramhat
+	name = "Singing Telegram Suit"
+	icon = 'icons/obj/custom.dmi'
```
Comment (Ghommie) [inline modular_citadel/code/modules/custom_loadout/custom_items.dm]: Your item icon_states aren't actually residing in this file.

File: modular_citadel/code/modules/custom_loadout/custom_items.dm [hunk 4]
```diff
@@ -439,3 +436,24 @@
 	icon_state = "moveralls"
 	mutantrace_variation = NO_MUTANTRACE_VARIATION
 
+/obj/item/clothing/head/telegramhat
+	name = "Singing Telegram Hat"
+	icon = 'icons/obj/custom.dmi'
```
Comment (Ghommie) [inline modular_citadel/code/modules/custom_loadout/custom_items.dm]: idem

File: modular_citadel/code/modules/custom_loadout/custom_items.dm [hunk 5]
```diff
@@ -439,3 +436,24 @@
...
+	alternate_worn_icon = 'icons/mob/custom_w.dmi'
```
Comment (Ghommie) [inline modular_citadel/code/modules/custom_loadout/custom_items.dm]: idem.
Your worn icon states are actually located in a rather outdated version of `modular_citadel/icons/obj/clothing/turtlenecks.dmi` it seems, they should be moved elsewhere. And make sure the file is up to date with the upstream.

File: modular_citadel/code/modules/custom_loadout/custom_items.dm [hunk 6]
```diff
@@ -439,3 +436,24 @@
...
+	desc = "A red fez like hat, has a yellow lining on the side."
+	item_state = "telegramhat"
+	icon_state = "telegramhat"
+	armor = list("melee" = 0, "bullet" = -500, "laser" = 0, "energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 0, "acid" = 0) //Dont shoot the messager!
+	mutantrace_variation = NO_MUTANTRACE_VARIATION
+
+/obj/item/clothing/under/telegramhat
+	name = "Singing Telegram Suit"
+	icon = 'icons/obj/custom.dmi'
+	alternate_worn_icon = 'icons/mob/custom_w.dmi'
```
Comment (Ghommie) [inline modular_citadel/code/modules/custom_loadout/custom_items.dm]: Same.

File: modular_citadel/code/modules/client/loadout/__donator.dm (hunk 1 above)
Comment (Trilbyspaceclone) [reply modular_citadel/code/modules/client/loadout/__donator.dm]: That is my ckey

File: modular_citadel/code/modules/custom_loadout/custom_items.dm (hunk 3 above)
Comment (Trilbyspaceclone) [reply modular_citadel/code/modules/custom_loadout/custom_items.dm]: Dose this fix it?

File: modular_citadel/code/modules/custom_loadout/custom_items.dm (hunk 4 above)
Comment (Trilbyspaceclone) [reply modular_citadel/code/modules/custom_loadout/custom_items.dm]: Woops

File: modular_citadel/code/modules/custom_loadout/custom_items.dm (hunk 5 above)
Comment (Trilbyspaceclone) [reply modular_citadel/code/modules/custom_loadout/custom_items.dm]: Its a turtleneck so I added it their

**Review by silicons — DISMISSED** [review]

File: modular_citadel/code/modules/custom_loadout/custom_items.dm [hunk 7]
```diff
@@ -439,3 +436,23 @@
 	icon_state = "moveralls"
 	mutantrace_variation = NO_MUTANTRACE_VARIATION
 
+/obj/item/clothing/head/telegramhat
+	name = "Singing Telegram Hat"
+	icon = 'modular_citadel/icons/obj/clothing/cit_hats.dmi'
```
Comment (Ghommie) [inline modular_citadel/code/modules/custom_loadout/custom_items.dm:441]: It's still missing the alternate_worn_icon, That's located in `modular_citadel/icons/obj/clothing/turtlenecks.dmi` for some reason.

File: modular_citadel/code/modules/custom_loadout/custom_items.dm [hunk 8]
```diff
@@ -439,3 +436,23 @@
...
+	item_state = "telegramhat"
+	icon_state = "telegramhat"
+	armor = list("melee" = 0, "bullet" = -500, "laser" = 0, "energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 0, "acid" = 0) //Dont shoot the messager!
+	mutantrace_variation = NO_MUTANTRACE_VARIATION
+
+/obj/item/clothing/under/telegramhat
+	name = "Singing Telegram Suit"
+	icon = 'modular_citadel/icons/obj/clothing/cit_clothes.dmi'
+	alternate_worn_icon = 'modular_citadel/icons/obj/clothing/turtlenecks.dmi'
+	desc = "'Hello, I am, your singing, telegram!' \
```
Comment (Ghommie) [inline modular_citadel/code/modules/custom_loadout/custom_items.dm:453]:
```suggestion
	desc = "'Hello, I am your singing telegram!' \
```
Unless it's about phonetic pauses.

File: modular_citadel/code/modules/custom_loadout/custom_items.dm [hunk 9]
```diff
@@ -439,3 +436,23 @@
...
+				A red button up suit that has a the insignia of a out dated means of relaying information to people \
```
Comment (Ghommie) [inline modular_citadel/code/modules/custom_loadout/custom_items.dm]:
```suggestion
				A red button up suit bearing the insignia of an outdated mean of relaying information to people. \
```

**Review by Ghommie — CHANGES_REQUESTED** [review]
`modular_citadel/icons/obj/clothing/turtlenecks.dmi` is still fouled up by these changes. Apparently a bunch of item sprites for sweaters and the CMO turtleneck are affected by this. Move your hat and suit worn sprites to `icons/mob/head.dmi` and `icons/mob/uniform.dmi` respectively.

File: modular_citadel/code/modules/custom_loadout/custom_items.dm (hunk 8 above)
Comment (Trilbyspaceclone) [reply modular_citadel/code/modules/custom_loadout/custom_items.dm:453]: In the movie is phonetic pauses so ya <3

File: modular_citadel/code/modules/client/loadout/__donator.dm [hunk 10]
```diff
@@ -373,4 +373,19 @@ datum/gear/darksabresheath
...
```
Comment (deathride58) [inline modular_citadel/code/modules/client/loadout/__donator.dm]: its case sensitive, all letters have to be lowercase

**Review by deathride58 — CHANGES_REQUESTED** [review]
It's up to Jay whether he wants to allow this or not

**Comment by silicons** [comment]
inactivity

**Pull request closed without merging**